tests/visco_plastic_differentplasticitydifferentphases.prm

```diff
@@ -0,0 +1,105 @@
+# Copy of visco_plastic_phases.prm, but tests different plasticity parameters for different phases.
```

Comment:
I would consider renaming the file to something simpler like `visco_plastic_phases_plasticity` or `visco_plastic_phases_plastic_parameters`?